dev/test/specs/starter-test.js

```diff
@@ -0,0 +1,192 @@
+/*
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+let assert = require('assert');
```

Comment:
We had to use the other form for our other browser tests, like:

`let assert = chai.assert`

Why is this ok here, maybe because this runs within Selenium rather than Mocha in a browser? Is this requiring the Chai assert lib or something else?

Reply:
Great question.

The require() call is specific to node. Because these are selenium tests, I don't think they can run in the browser, so this style is ok here.

I tried the other styles, both

let assert = chai.assert

and

import chai from 'chai';

One of those would be nice for consistency, but neither seems to work.